lms/templates/verify_student/missed_deadline.html

```diff
@@ -16,15 +16,27 @@
 </%block>
 
 <%block name="content">
-<section class="outside-app">
-  <p>
-
-  % if deadline_name == PayAndVerifyView.VERIFICATION_DEADLINE:
-  ${_(u"The verification deadline for {course_name} was {date}. Verification is no longer available.").format(
-      course_name=course.display_name, date=deadline)}
-  % elif deadline_name == PayAndVerifyView.UPGRADE_DEADLINE:
-  ${_(u"The deadline to upgrade to a verified certificate for this course has passed.")}
-  % endif
-  </p>
-</section>
+    <section class="outside-app">
+        <p
+            class="localized-datetime"
+            data-datetime="${deadline}"
+            data-timezone="${user_timezone}"
+            data-language="${user_language}"
+            % if deadline_name == PayAndVerifyView.VERIFICATION_DEADLINE:
+            data-string="${
+                _(
+                    u'The verification deadline for {course_name} was {{date}}. Verification is no longer available.'
+                ).format(
+                    course_name=course.display_name
+                )
```

Comment:
Oh bad ass, I think earlier I did a '{date}' in the format, but yours is better!